mindsdb/integrations/handlers/notion_handler/README.md

```diff
@@ -0,0 +1,56 @@
+# Notion Handler
+
+This is the implementation of the Notion handler for MindsDB.
+
+## Notion
+Notion is a note-taking and productivity freemium cloud platform.
+In short, notion has all-in-one workspace tool that integrates kanban boards, tasks, wikis, and database.
+In this handler. python client of api is used and more information about this python client can be found (here)[https://pypi.org/project/notion-client/]
+
+
+## Implementation
+This handler was implemented as per the MindsDB API Handler documentation.
+
+
+The required arguments to establish a connection are,
+* `notion_api_token`: API key for acessing the Notion instance.
+
+
+## Usage
+In order to make use of this handler and connect to an Notion in MindsDB, the following syntax can be used,
+~~~~sql
+CREATE DATABASE notion_source
+WITH
+engine='notion',
+parameters={
+    "notion_api_token": "<notion-api-token>",
+};
+~~~~
+
+## Implemented Features
```

Comment:
Hey @Mr-Destructive,
Thank you for this PR. Could you please add how to retrieve the IDs for each of these components (databases, pages and blocks)? I am assuming it is from the URL, but it might be good to include it for users who are not aware.
I think it might also be a good idea to explain what each of these are in maybe a single sentence. For instance, a database is a collection of pages. I actually do not know what a block is.